config/clients/go/template/configuration.mustache

```diff
@@ -18,12 +18,12 @@ type RetryParams struct {
 
 // Configuration stores the configuration of the API client
 type Configuration struct {
-    ApiScheme      string                   `json:"apiScheme,omitempty"`
-    ApiHost        string                   `json:"apiHost,omitempty"`
-    StoreId        string                   `json:"storeId,omitempty"`
-    Credentials    *credentials.Credentials `json:"credentials,omitempty"`
-    DefaultHeaders map[string]string        `json:"defaultHeader,omitempty"`
-    UserAgent      string                   `json:"userAgent,omitempty"`
+	ApiScheme      string                   `json:"api_scheme,omitempty"`
```

Comment:
Is there any reason we change the spacing?

Reply:
Don't worry about the spacing, go fmt handles fixing it after generation (so in the generated code, the spacing will be correct).

In this case the updated spacing is "correct", the older spacing was being formatted away